src/modules/lodash.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable import/prefer-default-export */
+import _ from 'lodash';
+
```

Comment:
- [ ] Erase the commented lines that indicate that linters are disabled. You are expected to use linters to ensure your code is clean and maintainable